feature/path-tracking/src/main/java/com/mifos/feature/path_tracking/PathTrackingScreen.kt

```diff
@@ -0,0 +1,317 @@
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat
+import androidx.core.content.ContextCompat.registerReceiver
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.isGranted
+import com.google.accompanist.permissions.rememberMultiplePermissionsState
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import com.google.maps.android.compose.GoogleMap
+import com.google.maps.android.compose.MapUiSettings
+import com.google.maps.android.compose.rememberCameraPositionState
+import com.mifos.core.common.utils.Constants
+import com.mifos.core.designsystem.component.MifosCircularProgress
+import com.mifos.core.designsystem.component.MifosScaffold
+import com.mifos.core.designsystem.component.MifosSweetError
+import com.mifos.core.designsystem.icon.MifosIcons
+import com.mifos.core.designsystem.theme.Black
+import com.mifos.core.designsystem.theme.White
+import com.mifos.core.objects.user.UserLatLng
+import com.mifos.core.objects.user.UserLocation
+
+@Composable
+fun PathTrackingScreen(
+    userId: Int,
+    onBackPressed: () -> Unit,
+    onPathTrackingClick: (List<UserLatLng>) -> Unit
+) {
+
+    val context = LocalContext.current
+    val viewModel: PathTrackingViewModel = hiltViewModel()
+    val state by viewModel.pathTrackingUiState.collectAsStateWithLifecycle()
+    val refreshState by viewModel.isRefreshing.collectAsStateWithLifecycle()
+    val userStatus by viewModel.userStatus.collectAsStateWithLifecycle()
+
+    DisposableEffect(Unit) {
+
+        val notificationReceiver = object : BroadcastReceiver() {
```

Comment:
Why are you using BroadcastReceiver

Reply:
This was previously implemented. it receives datas from path tracking service, but right now path service is not working, so for now, I simply added this so we don't forget this implementation. once path tracking service is fixed, if this receiver is no longer useful, I'll remove it.

Reply:
Create a ticket to migrate service in workmanager